source/documentation/components/mot-history.md

```diff
@@ -1,2 +1,3 @@
-## MOT History
+## MOT Intelligence
+
```

Comment:
This is mot-history file and mot intelligence file needs to be created but I'll still merge this.